src/Joonasw.AspNetCore.SecurityHeaders/ReportToOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Joonasw.AspNetCore.SecurityHeaders
+{
+    public class ReportToOptions
+    {
+        public string GroupMemeberName { get; set; }
```

Comment:
The name here seems to have a typo. Also all the members should have a `[JsonProperty("prop-name")]` to ensure they get serialized with the correct names.

Reply:
Good spot. It crossed my mind and then got forgotten. I'll get it added to all of them.